src/splat.F

```diff
@@ -18,34 +18,29 @@
 C> if an equator point exists, its weight should be halved.
 C> This version invokes the ibm essl matrix solver.
 C>
-C> PROGRAM HISTORY LOG:
-C> -  96-02-20  IREDELL
-C> -  97-10-20  IREDELL  ADJUST PRECISION
-C> -  98-06-11  IREDELL  GENERALIZE PRECISION USING FORTRAN 90 INTRINSIC
-C> - 1998-12-03  IREDELL  GENERALIZE PRECISION FURTHER
-C> - 1998-12-03  IREDELL  USES AIX ESSL BLAS CALLS
-C> - 2009-12-27  DSTARK   updated to switch between ESSL calls on an AIX
+C> ### Program History Log
+C> Date | Programmer | Comments
+C> -----|------------|---------
+C> 96-02-20 | IREDELL |
+C> 97-10-20 | IREDELL | ADJUST PRECISION
+C> 98-06-11 | IREDELL | GENERALIZE PRECISION USING FORTRAN 90 INTRINSIC
+C> 1998-12-03 | IREDELL | GENERALIZE PRECISION FURTHER
+C> 1998-12-03 | IREDELL | USES AIX ESSL BLAS CALLS
+C> 2009-12-27 | DSTARK | updated to switch between ESSL calls on an AIX
 C>                      platform, and Numerical Recipies calls elsewise.
-C> - 2010-12-30  SLOVACEK update alignment so preprocessor does not cause
+C> 2010-12-30 | SLOVACEK | update alignment so preprocessor does not cause
 C>                      compilation failure
-C> - 2012-09-01  E.Mirvis & M.Iredell merging & debugging linux errors 
+C> 2012-09-01 | E.Mirvis & M.Iredell | merging & debugging linux errors 
 C>			of _d and _8 using generic LU factorization.   
-C> - 2012-11-05  E.Mirvis  generic FFTPACK and LU lapack were removed
+C> 2012-11-05 | E.Mirvis | generic FFTPACK and LU lapack were removed
 C>
-C> @param IDRT     - INTEGER GRID IDENTIFIER
-C>                (IDRT=4 FOR GAUSSIAN GRID,
-C>                 IDRT=0 FOR EQUALLY-SPACED GRID INCLUDING POLES,
-C>                 IDRT=256 FOR EQUALLY-SPACED GRID EXCLUDING POLES)
-C> @param JMAX     - INTEGER NUMBER OF LATITUDES.
-C> @param[out] SLAT - REAL (JMAX) SINES OF LATITUDE.
-C> @param[out] WLAT - REAL (JMAX) GAUSSIAN WEIGHTS.
-C>
-C> SUBPROGRAMS CALLED:
-C>  - dgef()         MATRIX FACTORIZATION - ESSL
-C>  - dges()         MATRIX SOLVER - ESSL
-C>  - ludcmp()       LU factorization - numerical recipies
-C>  - lubksb()       Matrix solver - numerical recipies
-C>      
+C> @param IDRT GRID IDENTIFIER
+C>             (IDRT=4 FOR GAUSSIAN GRID,
```

Comment:
Change to use bullet list, like this:

```
C> @param IDRT GRID IDENTIFIER
C>  - IDRT=4 FOR GAUSSIAN GRID,
C>  - IDRT=0 FOR EQUALLY-SPACED GRID INCLUDING POLES,
C>   - IDRT=256 FOR EQUALLY-SPACED GRID EXCLUDING POLES

```

Note that each line starts with "- " which will cause doxygen to create a bullet-list.

Note also extra spaces have been removed.